userguide/system.rst

```diff
@@ -299,28 +298,29 @@ explains these options in more detail.
    +-------------+-----------+----------------------------------------------------------------------------------------------------+
-   | Prefer      | checkbox  | should only be used for NTP servers that are known to be highly accurate, such as those with       |
-   |             |           | time monitoring hardware                                                                           |
+   | Prefer      | checkbox  | This option is only recommended for highly accurate NTP servers, such as those with                |
+   |             |           | time monitoring hardware.                                                                          |
+   |             |           |                                                                                                    |
    +-------------+-----------+----------------------------------------------------------------------------------------------------+
-   | Min. Poll   | integer   | power of 2 in seconds; cannot be lower than                                                        |
-   |             |           | *4* or higher than :guilabel:`Max. Poll`                                                           |
+   | Min. Poll   | integer   | Power of 2 in seconds. Cannot be lower than                                                        |
+   |             |           | *4* or higher than :guilabel:`Max. Poll`.                                                          |
    |             |           |                                                                                                    |
    +-------------+-----------+----------------------------------------------------------------------------------------------------+
-   | Max. Poll   | integer   | power of 2 in seconds; cannot be higher than                                                       |
-   |             |           | *17* or lower than :guilabel:`Min. Poll`                                                           |
+   | Max. Poll   | integer   | Power of 2 in seconds. Cannot be higher than                                                       |
+   |             |           | *17* or lower than :guilabel:`Min. Poll`.                                                          |
    |             |           |                                                                                                    |
    +-------------+-----------+----------------------------------------------------------------------------------------------------+
-   | Force       | checkbox  | forces the addition of the NTP server, even if it is currently unreachable                         |
+   | Force       | checkbox  | Forces the addition of the NTP server, even if it is currently unreachable.                        |
    |             |           |                                                                                                    |
```

Comment:
s/Forces the addition of the NTP server, even if it is currently unreachable./Force the addition of an NTP server even if currently unreachable./